CommonPackageConfig.cmake

```diff
@@ -63,6 +63,13 @@ if(CMAKE_MAJOR_VERSION LESS 3 OR NOT TARGET ${PROJECT_NAME})
   return()
 endif()
 
+# A project may want to include CommonCPack without having used common_library.
+# Returning in that case (Probably we don't event want to generate other
+# Config files since there's no library).
+if(NOT ${PROJECT_NAME}Targets)
```

Comment:
This does not work for me. Even if a project adds a target to the export set aka ${PROJECT_NAME}Targets, it early outs here. To my understanding, ${PROJECT_NAME}Targets is not a variable you can check like this.